src/CustomItem.cs

```diff
@@ -11,12 +18,12 @@ public class CustomItem
 	public string Name;
 	public string Description;
 
-	private readonly GameObject ItemPrefab;
-	public GameObject WorldObject {get; private set;}
+	public GameObject ItemPrefab { get; private set; }
 
 	public InventoryItemSpec ItemSpec => ShopData.item;
-	// public ShelfItem ShelfComponent => ShopData.shelfItem;
 	public ShopItemData ShopData { get; private set; }
+
+	public GameObject ProvidedPrefab { get; private set; }
 	
 	
 	/// <param name="name">name of the item</param>
```

Comment:
could you remove the parameters that don't exist anymore from these comments?

Reply:
Done - also added the one replaced param and fixed the name on the one changed param